src/Page/TextLayer.css

```diff
@@ -94,6 +81,11 @@
   background-color: var(--highlight-selected-bg-color);
 }
 
+.textLayer ::selection {
+  background: blue;
+  background: AccentColor;
+}
```

Comment:
This block was intentionally removed - avoids custom selection color which serves no good purpose